frontend/src/screens/apps/Index.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable react-refresh/only-export-components */
+import { useEffect, useState } from "react";
+import { useNavigate } from 'react-router-dom';
+
+import Loading from '../../components/Loading';
+import { App, NostrEvent, ListAppsResponse } from "../../types";
+
+function Connections() {
+  const [apps, setApps] = useState<App[] | null>(null);
+  const [eventsCounts, setEventsCounts] = useState<Record<App["id"], number> | null>(null);
+  const [lastEvents, setLastEvents] = useState<Record<App["id"], NostrEvent> | null>(null);
+  const navigate = useNavigate();
+
+  const handleRowClick = (appId: App["nostrPubkey"]) => {
```

Comment:
can the row not be a standard link? I try to avoid these because they are poor for accessibility